HW1.java

```diff
@@ -0,0 +1,13 @@
+public class HW1 {
+
+        int Year = 2023;
+
+        static void myClass(int a, int b) {
+            int sum = a + b; //sum of 2 numbers
```

Comment:
Please remove the comment because it's extra here
By method name we can understand what this method does